packages/smooth_app/lib/pages/crop_page.dart

```diff
@@ -204,11 +208,18 @@ class _CropPageState extends State<CropPage> {
                         alwaysMove: true,
                       ),
                     ),
-                    Center(
-                      child: EditImageButton(
-                        iconData: Icons.send,
-                        label: appLocalizations.send_image_button_label,
-                        onPressed: () async => _mayExitPage(saving: true),
+                    Padding(
+                      padding: const EdgeInsets.symmetric(
+                        horizontal: VERY_SMALL_SPACE,
+                        vertical: SMALL_SPACE,
```

Comment:
Not sure it's a good idea to add a padding there, as it means less space for the image, that is already not that big.